client/src/pages/ItemList/item-list.jsx

```diff
@@ -1,24 +1,52 @@
 import React, { useState, useEffect } from "react";
 import { ProductItems } from "../../components/ProductItems/ProductItems.jsx";
-import data from "../../data/data.json";
+import groceryData from "../../data/grocery-data.json";
+import { v4 as uuidv4 } from "uuid";
+import styles from "./ItemList.module.scss";
 
 export const ItemList = () => {
 	const [products, setProducts] = useState([]);
-	const [data, setData] = useState(null);
+	const [jsonData, setJsonData] = useState(null);
+
+	// useEffect(() => {
+	// 	const fetchData = async () => {
+	// 		try {
+	// 			const response = await fetch(groceryData);
+	// 			const jsonData = await response.json();
+	// 			setData(jsonData);
+	// 		} catch (error) {
+	// 			console.error("Error:", error);
+	// 		}
+	// 	};
+	// 	fetchData();
+	// }, []);
 
 	useEffect(() => {
-		const fetchData = async () => {
-			try {
-				const response = await fetch(data);
-				const jsonData = await response.json();
-				setData(jsonData);
-			} catch (error) {
-				console.error("Error:", error);
-			}
-		};
-		fetchData();
-	}, []);
+		if (groceryData) {
+			const productsWithIds = groceryData.map((product) => ({
+				...product,
+				id: uuidv4()
+			}));
+			setProducts(productsWithIds);
+		}
+	}, [jsonData]);
+
+	console.log(products);
```

Comment:
Nicht vergesen das zu entfernen. Nur für die Entwicklung wichtig, in der Produktion wollen wir das nicht haben.
```suggestion
```